contracts/polygon/connectors/limitOrder/interface.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity ^0.8.1;
+// SPDX-License-Identifier: MIT
+
+interface UniLimitOrder {
```

Comment:
```suggestion
interface IUniLimitOrder {
```

Reply:
Follow a pattern. In the interface always add `I` in beginning or `Interface` in the end. Makes it easy to separate them from contracts. After updating this you will need to update things at many all the places you are using it.